Task 2/Task 2.1/CustomStringLibrary/CustomString.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace CustomStringLibrary
+{
+    /*
+     * Напишите собственный класс, описывающий строку как массив символов. Реализуйте для этого
+     * класса типовые операции (сравнение, конкатенация, поиск символов, конвертация из/в массив
+     * символов). Подумайте, какие функции вы бы добавили к имеющемуся в .NET функционалу строк
+     * (достаточно 1-2 функций).
+     * 
+     * Вариант со * - подумайте над использованием в своем классе функционала индексатора
+     * (indexer). Реализуйте его для своей строки.
+     * 
+     * Вариант с ** - попробуйте создать из своей сборки переносимую библиотеку (DLL). Осмысленно
+     * назовите её, а также namespace и сам класс. Попробуйте использовать написанный вами класс в
+     * другом проекте.
+     */
+    public class CustomString
+    {
+        private char[] _content;
+
+        public int Length { get => _content.Length; }
```

Comment:
Sorry, didn't leave a comment before, but here you can do this (practically the same thing):
```suggestion
        public int Length => _content.Length;
```